mobile/app/(app)/tosignin/index.tsx

```diff
@@ -28,7 +27,7 @@ export default function Page() {
       try {
         if (!vaults || vaults.length === 0) {
           await dispatch(fetchVaults()).unwrap()
-          dispatch(checkForKeyOnChains()).unwrap()
+          // dispatch(checkForKeyOnChains()).unwrap()
```

Comment:
Is it normal that it's commented? If yes, can you remove it?
Or you forgot to uncomment the line?